src/rtScriptDuk/debugger/duk_debugger_interface_unix.c

```diff
@@ -0,0 +1,266 @@
+#include <stdio.h>
```

Comment:
Let's rename to duk_debugger_interface.c and add it to rtScriptDuk/debugger/unix/.